tb_db/models.py

```diff
@@ -59,17 +60,39 @@ class Sample(Base):
     accession = Column(String)
     collection_date = Column(Date)
 
-    cgmlst_cluster = relationship("CgmlstCluster", secondary=association_table_cgmlst, backref = 'samples')
-    miru_cluster = relationship("MiruCluster", secondary=association_table_miru, backref='samples')
+    library = relationship("Library", backref = 'samples')
+    miru_profile = relationship("MiruProfile", backref = 'samples', cascade="all,delete")
+    miru_cluster = relationship("MiruCluster", secondary=association_table_miru, backref ='samples', cascade="all, delete")
+
 
 
 class Library(Base):
     """
     """
 
     sample_id = Column(Integer, ForeignKey("sample.id"), nullable=False)
+    sample_name = Column(String)
```

Comment:
Does `sample_name` belong to the `Sample` entity? There's a `library_id` already. What's the difference between `sample_name` and `library_id`?

Reply:
yea so sample_name is the original sample name with all the suffixes left in, and sample_id is the ones with suffixes stripped off and linked to samples table by foreign keys. I am not sure what library_id does, they are just nulls now, and they were there since the beginning. I can take them out.